presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -379,6 +380,8 @@ public CounterStat getHighMemorySplitSource()
                     partitions);
 
             ImmutableList.Builder<HivePartitionMetadata> results = ImmutableList.builder();
+            int numOfPartitions = 0;
+            ArrayList<String> partitionsNotReadable = new ArrayList<String>(); 
```

Comment:
use ImmutableList.Builder

Reply:
@highker Thanks for the feedback. Are you saying I should change the partitionsNotReadable's type to Immutable.Builder<String>? partitionsNotReadable just holds the names of the partitions that are not readable. Considering that partName is of type String, I figured that ArrayList<String> would be appropriate.